Contest_API.md

```diff
@@ -268,14 +268,20 @@ Properties for file reference objects:
 The `href` property may be an [absolute or relative
 URL](https://tools.ietf.org/html/rfc3986); relative URLs must be
 interpreted relative to the `baseurl` of the API. For example, if
-`baseurl` is <https://example.com/api>, then the following are
+`baseurl` is <https://example.com/api/>, then the following are
 equivalent JSON response snippets pointing to the same location:
 
 ```json
   "href":"https://example.com/api/contests/wf14/submissions/187/files"
   "href":"contests/wf14/submissions/187/files"
+  "href":"/api/contests/wf14/submissions/187/files"
 ```
 
+The trailing slash in the `baseurl` is important! If `baseurl` is
+<https://example.com/api/foobar>, then the relative URL
+`contests/wf14/submissions/187/files` *also* resolves to
+`https://example.com/api/contests/wf14/submissions/187/files`.
```

Comment:
What about:

```suggestion
Note that the `baseurl` always ends in a slash.
```

and move it to around line 51.

Reply:
Updated.